pwndbg/lldblib/__init__.py

```diff
@@ -0,0 +1,9 @@
+from __future__ import annotations
```

Comment:
Does this need to be in this PR or can we move it to the other one?

Reply:
D'oh. Forgot to remove that one when I was splitting the commit into two. Thanks for pointing it out, I'll take it out.